cartographer_ros/cartographer_ros/node.cc

```diff
@@ -448,6 +448,11 @@ void Node::HandleOdometryMessage(const int trajectory_id,
                                  const string& sensor_id,
                                  const nav_msgs::Odometry::ConstPtr& msg) {
   carto::common::MutexLocker lock(&mutex_);
+  auto sensor_bridge_ptr = map_builder_bridge_.sensor_bridge(trajectory_id);
```

Comment:
Either inline or also use on line 456. Also, these pointers should be const, no?